src/modules/ekf2/EKF2.cpp

```diff
@@ -1002,7 +1002,10 @@ void EKF2::PublishInnovations(const hrt_abstime &timestamp)
 	_estimator_innovations_pub.publish(innovations);
 
 	// calculate noise filtered velocity innovations which are used for pre-flight checking
-	if (!_ekf.control_status_flags().in_air) {
+	vehicle_status_s vehicle_status;
```

Comment:
Can we revert this part now that preflight check isn't directly factored into z and vz validity flags? The result of the preflight check is optionally used by the centralized estimator checks only when disarmed.

Reply:
done